### DISCUSSION TITLE
Home Assistant su Docker non vede Sonoff Dongle-E

### DISCUSSION THREAD

**Discussion by TimVoda** [indomus/forum#2352] [HUB: Home Assistant]
Buongiorno a tutti,
 ho Home Assistant su Debian/Docker su minipc, il tutto installato seguendo le guide di Indomus.
Il Sonoff dongle E viene visto dal SO, al punto che ho aggiornato con successo il FW.
Non viene trovato da ZHA su Docker.
Il dongle in sè funziona, perchè su di un' altra installazione (Raspberry Pi/Docker) faceva tutto quello che ci si aspettava da lui.
Qualche suggerimento?
Grazie a chi vorrà rispondere

**Comment by snakuzzo** [comment]
Devi mappare la porta su cui è il dongle quando crei il container

**Reply by TimVoda** [reply]
Ciao e grazie per la risposta
aggiungendo 

```
    devices:
    - /dev/ttyACM0:/dev/ttyACM0

```

allo .yaml?
o mi sono perso qualcosa?

**Reply by TimVoda** [reply]
Ho aggiunto queste righe e verificato col vostro script che fosse effettivamente il percorso del dongle

**Reply by snakuzzo** [reply]
E ?

**Reply by snakuzzo** [reply]
Se e usi zigbee2mqtt, devi anche assicurarti che la parte alla destra dei due punti sia uguale nel configuration.yaml di zigbee2mqtt

**Reply by TimVoda** [reply]
Non mi sono spiegato, temo.
Uso (userei) ZHA, non zigbee2MQTT.
Le righe di cui sopra sono state aggiunte subito allo .yaml, ciò nonostante non sono mai riuscito a far vedere il dongle a ZHA.
E' per questo che ho pensato di chiedere aiuto; nella precedente installazione su Raspberry con lo stesso .yaml il dongle si vedeva e associava periferiche senza problemi

**Reply by snakuzzo** [reply]
Hai ricreato il container dopo aver modificato il file yaml?

**Reply by TimVoda** [reply]
appena posso provo e posto i risultati, intanto grazie.

**Reply by snakuzzo** [reply]
Visto che parli di yaml, mmagino tu stia usando `docker-compose`.
Quando fai una qualsiasi modifica al compose devi sempre ricreare il container.
Per farlo ti basta ridare il comando

```
docker-compose up -d
```

**Reply by TimVoda** [reply]
..ed era proprio quello..
..non so se marcare solved la conversazione o chiedere che venga cancellata.. :)
grazie delle risposte e soprattutto della pazienza

**Reply by snakuzzo** [reply]
Non preoccuparti. A posto così :)
